pepper-apis/src/main/java/org/broadinstitute/ddp/db/dao/JdbiUser.java

```diff
@@ -93,4 +93,18 @@ default UserDto findByUserId(long userId) {
 
     @SqlUpdate("delete from user where guid in (<guid>)")
     int deleteAllByGuids(@BindList(value = "guid", onEmpty = BindList.EmptyHandling.NULL) Set<String> guids);
+
+    @SqlQuery("SELECT u.*"
```

Comment:
Although not used in this PR, at one point I needed a way to pull all enrolled participants for a study, so I'd like to leave this here for general use.